aequilibrae/paths/assignment_paths.py

```diff
@@ -0,0 +1,119 @@
+import os
+from typing import Dict, List
+import numpy as np
+import pandas as pd
+from aequilibrae import logger
+from aequilibrae.project.database_connection import ENVIRON_VAR
+from aequilibrae.project.database_connection import database_connection
+
+
+# TODO: let's make it optional to keep path files in memory, although this can get out of control very quickly it should
+# be much quicker when working with a single origin
+
+# TODO: factor out AssignmentResultsTable into different file, also get rid of dirty eval bu changing creation
+
+# FIXME: this is for zone_index and compressed link ids
+# for link ids, look up what we are doing in graph - we might want to keep the order of links for a single compressed
+# link
+# for zones, we need to do the inverse of graph.compact_nodes_to_indices
+
+
+class TrafficClassIdentifier(object):
+    def __init__(self, name: str, id: str):
+        self.name = name
+        self.id = id
+
+
+class AssignmentResultsTable(object):
```

Comment:
I think some documentation is needed here

Reply:
Or not... If it is used by another class and not meant to be used directly... My bad...

Reply:
I think it would be good to have this available and documented for usability, but I'd say this is out-of-scope for this PR